src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,18 @@
+import { Person } from '../../types/Person';
+import { DropdownItem } from '../DropdownItem';
+
+interface Props {
+  people: Person[];
+}
+
+export const DropDownMenu: React.FC<Props> = ({ people }) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map(person => (
+          <DropdownItem person={person} key={person.name} />
```

Comment:
You are using `person.name` as a key for the `DropdownItem`. This is not recommended because names are not guaranteed to be unique. If there are two people with the same name, React will have trouble determining which elements in the list have changed, are added, or are removed. This can lead to unexpected behavior in your app. It's recommended to use unique and stable keys for list items, such as an id that won't change. If there is no unique id from the data, you might need to generate a unique id when the data is being processed. Here is a related article that explains why keys are important in React: [Don’t Use Index as Key for React List](https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052).